bin/powershell/metric-windows-cpu-load.ps1

```diff
@@ -1,18 +1,4 @@
 #
-#   metric-windows-cpu-load.ps1
```

Comment:
why are we removing this?

Reply:
To be honest, I don't know why the header is removed in the change. Wasn't and isn't my intention.